packages/travix-ui-kit/tests/unit/slidingPanel/slidingPanelHeader.spec.js

```diff
@@ -55,8 +70,23 @@ describe('SlidingPanel', () => {
       const title = <span className="my-class"> my label </span>;
       const renderTree = mount(
         <SlidingPanelHeader
-          leftBlock={<button data-rel="close" style={{ marginLeft: '15px' }}> ← </button>}
-          rightBlock={<button data-rel="close" style={{ marginRight: '15px' }}> close me! </button>}
+          leftBlock={
+            props => (
+              <button
+                {...props}
+                style={{ marginLeft: '15px' }}
+              > ← </button>
+            )
+          }
+          rightBlock={
```

Comment:
Should we also have a test using the default `rightBlock`?

Reply:
I'll add test for this case

Reply:
Updated